### DISCUSSION TITLE
Home Assistant App Integration

### DISCUSSION THREAD

**Discussion by fabioteixei** [goauthentik/authentik#7952] [Q&A]
Hello

The integration with home assistant works well when we use the browser, everyting is great.
But when we use the home assistant App on the phone, when we try to open the app we get the Home Assistant Auth error, then i click on retry and i'm forwarded to the authentication page but on the phone browser and not on the app, i authenticate on the browser and home assistent opens on the browser but not on the app. O the App the same retry screen remains.

Is there any way to work aroud that?

Thanks

**Comment by deviantintegral** [comment]
I don’t believe the mobile Home Assistant apps support external authentication at all.

https://github.com/home-assistant/android/issues/1438
https://github.com/home-assistant/iOS/issues/2112
